R/kiss-report.R

```diff
@@ -132,16 +143,44 @@ read.KissReport <- function(report) {
   # resultsLink should now be ready
   print("Report ready, pulling results")
   results <- as.data.frame(loadPages(report, resultsLink), stringsAsFactors = FALSE)
-  if (!is.null(report$columnNames)) {
-    names(results) <- report$columnNames
-  }
+  names(results) <- names(report)
```

Comment:
Add a comment:
# Set the names of the results to be "identity" + the labels used for the report calculations.